docs/csharp/cs-dev-kit-faq.md

```diff
@@ -71,6 +71,16 @@ Note that, depending on your network speed, installing the .NET Core runtime mig
 }
 ```
 
+### Error Acquiring .NET 7
+
+![Error Acquiring .NET 7](images/faq/error-acquiring-dotnet-7.png)
+
+Note: If you are located in China the Great Firewall of China may be blocking your .NET download which can cause a timeout.
+
+You will need to make sure you have .NET 7 installed. As a workaround, you can point the .NET runtime acquisition extension to an existing .NET 7 install:
+
+![Point the .NET runtime acquisition extension to an existing .NET 7 install](images/faq/point-dotnet-runtime-to-existing-dotnet-install.png)
```

Comment:
Better to use a code block for the settings JSON example than a screenshot.
That way users can easily copy and paste to get started